db/migrate/20240806130123_backfill_disbursement_position.rb

```diff
@@ -14,13 +14,13 @@ def up
     # OR submission version with any disbursements
     # SubmissionVersion.where("application ->'disbursements' IS NOT NULL")
 
-    claims = Submission.where(application_type: "crm7")
+    # claims = Submission.where(application_type: "crm7")
```

Comment:
Why not removed?

Reply:
I had a notion it would be helpful to see what the migration _used_ to do, but the more I think about it the more that's nonsense. I'll remove this (and the one below)

Reply:
There is some logic to maybe keeping the empty migrations around, I don't know how Rails handles "missing" migrations. I assume it handles cleanly and if so probably best just removing them yeah